src/PersonRow.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export const PersonRow = ({ peoples }) => (
+  <>
+    {peoples.map(people => (
+      <tr key={people.slug}>
+        <td>{people.name}</td><td>{people.sex}</td>
+        <td>{people.born}</td><td>{people.died}</td>
+        <td>{people.motherName}</td><td>{people.fatherName}</td>
```

Comment:
You should not only show the motherName and fatherName in the table. Task description says:

> Find a mother and a father by motherName and fatherName and add them to the person for future use

Every object in your `people` array should contain the `father` and the `mother` object. And then you need to take mother.name or father.name to show it in you table